metaflow/plugins/metadata_providers/service.py

```diff
@@ -39,6 +34,23 @@ def __init__(self, msg, http_code=None, body=None):
 class ServiceMetadataProvider(MetadataProvider):
     TYPE = "service"
 
+    _session = requests.Session()
+    _session.mount(
+        "http://",
+        requests.adapters.HTTPAdapter(
+            pool_connections=20,
```

Comment:
how did you determine this number?

Reply:
anecdotal -adjusting it higher or lower has no further practical impact.